kolibri/content/api.py

```diff
@@ -37,7 +38,7 @@
 
 
 logger = logging.getLogger(__name__)
-
+stemmer = Porter2Stemmer()
```

Comment:
This seems a bit risky because it assumes the search queries are in English. Results could be unpredictable when applied to other languages